build.rs

```diff
@@ -438,7 +438,10 @@ fn build_unix(target_info: &TargetInfo, out_dir: PathBuf) {
                   out_dir.clone(),
                   test_header_change);
 
-    let libcxx = if target_info.target_vendor() == "apple" ||
+    let libcxx = if target_info.target_vendor()
+        .map(|v| v == "apple")
```

Comment:
The problem here is that target_vendor returns only a valid value if target_build_utils is build with a nightly rustc. See the corresponding [issue](https://github.com/nagisa/target_build_utils.rs/issues/1) for more information. Therefore this function returns an Option. If the value is there we could check if it is apple otherwise we need to check that the os is ios or macos. We could also use only the second check for all variants, if you prefere.

(I've only a linux machine to test things)